src/main/java/com/example/user_management/api/converter/UserModelConverter.java

```diff
@@ -0,0 +1,28 @@
+package com.example.user_management.api.converter;
+
+import com.example.user_management.api.model.request.LoginRequestModel;
+import com.example.user_management.dto.LoginUserDto;
+import com.example.user_management.dto.RegisterUserDto;
+import com.example.user_management.api.model.request.RegisterUserRequestModel;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class UserModelConverter {
```

Comment:
There is single usage of UserModel like class, why it's called UserModelConverter?